### DISCUSSION TITLE
YUM repository

### DISCUSSION THREAD

**Discussion by DevDef** [dbeaver/dbeaver#22282] [Ideas]
As a Debian repository is provided, a YUM repository for the RPM packages would be very nice too, in particular to ease updates to newer versions of DBeaver. This is really easy to do, even with a Debian server (see @[here](https://www.cs.rug.nl/~jurjen/ApprenticesNotes/Creating_a_yum_repository_under_Debian.html)).

**Comment by Hubbitus** [comment]
Good idea. I'm also willing to see it. Could you please add it?
